src/java.base/share/classes/java/text/DecimalFormatSymbols.java

```diff
@@ -188,6 +188,17 @@ public static final DecimalFormatSymbols getInstance(Locale locale) {
         return dfsyms;
     }
 
+    /**
+     * Gets the locale used to create this instance.
```

Comment:
Nit: Since `@return` has the same description, `{@return ...}` can be used here.